package.json

```diff
@@ -96,6 +96,7 @@
     "dotenv": "^16.3.1",
     "gettext-parser": "1.3.0",
     "husky": "^1.3.1",
+    "imask": "^7.6.1",
```

Comment:
Why? You don't import this anywhere. 
Honestly though, doing this and an import in the js seems less impactful than creating a new asset class to import from the `unpkg` URL. (and it version locks!)